seda/assetlist.json

```diff
@@ -0,0 +1,38 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "seda",
+  "assets": [
+    {
+      "description": "The SEDA Token (SEDA), previously known as Flux Token (FLX), serves three distinct purposes: governance, participation, and network utilization.",
+      "denom_units": [
+        {
+          "denom": "aseda",
+          "exponent": 0
+        },
+        {
+          "denom": "seda",
+          "exponent": 18
+        }
+      ],
+      "base": "aseda",
+      "name": "SEDA",
+      "display": "seda",
+      "symbol": "SEDA",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/seda/images/seda.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/seda/images/seda.svg"
+      },
+      "coingecko_id": "seda-2",
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/seda/images/seda.png",
```

Comment:
either the uploaded images have to be visually identical, or these urls need to be in separate objects to show that they're different images.